test/integration/CMakeLists.txt

```diff
@@ -11,6 +11,7 @@ set(tests
   breadcrumbs.cc
   buoyancy.cc
   buoyancy_engine.cc
+  camera_lens_flare.cc
```

Comment:
can you move this line to the [`tests_needing_display`](https://github.com/gazebosim/gz-sim/blob/main/test/integration/CMakeLists.txt#L88) list below?